tests/func/test_checkout.py

```diff
@@ -410,7 +409,7 @@ def setUp(self):
 
     def test(self):
         with self._caplog.at_level(logging.INFO, logger="dvc"), patch.object(
-            sys, "stdout"
+            sys, "stderr"
         ) as stdout_mock:
```

Comment:
strictly speaking should update mentions of `stdout` to say `stderr` instead.

Reply:
@casperdcl please feel free to remove tests that you think are no longer needed.

Reply:
This is still named `stdout_mock` :) I would say we should stop testing output and rely on tqdm tests for that. Here we should test that `Tqdm` is called with appropriate params - much easier to mock and check.

Reply:
removed